README.md

```diff
@@ -185,23 +163,30 @@ var settings = {
 }
 
 // The rest is collections and can be stored in firestore
-var personCollection = [
+const personCollection = [
   {
     login: firebaseUserId, // Only for people with a login
     name: 'Test User',
     phone: '+62 123 123 123',
     address: 'Jalan Batu Mejan No. 88, Canggu, Kuta Utara, Kabupaten Badung, Bali 80361',
     email: 'some@address.com',
-    geolaction: {
+    geolocation: {
       latitude: '8.39111',
       longitude: '115.07361'
     },
-    type: 'employee | client | community manager | facility manager | super admin | village guy',
+    type: {
+      employee:false,
+      client:false,
+      community_manager:false,
+      facility_manager:false,
+      super_admin: true
```

Comment:
traditionally we prepend booleans with `is` or `has` to indicate they're bools

Reply:
unless there is overlap between these permissions, i think its okay to keep the `personCollection.type` set to a string rather than passing in a nested boolean. @piggydoughnut are you sure about this change? if you are, you have my full support, but it feels off